### PR TITLE
discord api --> announcements page

### PR DESCRIPTION
**Pull request by ashlxyzhang**
* made discord bot w/ these perms: Read Messages/View Channels, Read Message History & Message Content intent
* set up .env file with bot token & channel ids
* created api route in api/announcements
* set up test server to test get messages

im hungry now

### CODE REVIEW

**Comment by ashlxyzhang** [comment]
test

**Review by peicasey — APPROVED** [review]
Overall looks good :3 

Just lmk what the variable you decide to set localhost:3000 to so I can set it on the Netlify deployment + the small things i mentioned and we should be good!!

File: club-website/src/app/api/announcements/route.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
+export async function GET() {
+    const res = await fetch(`${process.env.DISCORD_API_ROUTE}/channels/${process.env.TAO_CHANNEL_ID}/messages`, {
```
[the remark this answers is not in the document]
Comment (ashlxyzhang) [reply club-website/src/app/api/announcements/route.ts]: no separate server, just runs through next api routes :3

[the remark this answers is not in the document]
Comment (peicasey) [reply club-website/src/app/api/announcements/route.ts]: yeah i realize i was tripping LOL for some reason i thought you had to run a separate server for the discord bot but i think that might just be for apps / i was bugging out

**Pull request merged**